tests/ignite/handlers/test_state_param_scheduler.py

```diff
@@ -13,142 +13,257 @@
 )
 
 
-def test_linear_scheduler_linear_increase_history():
+@pytest.mark.parametrize(
+    "max_epochs, initial_value, step_constant, step_max_value, max_value, save_history, expected_param_history",
+    [
```

Comment:
@fco-dv you can define configs just above and in a single line (if remove the last `,`) to avoid this expansion by black

Reply:
When calling black after defining configs as below without trailing `,` it is still making list expansion, maybe i'm missing something  ... Can we use `#fmt: off ... #fmt: on ` ?
```
@pytest.mark.parametrize(
    "max_epochs, initial_value, step_constant, step_max_value, max_value, save_history, expected_param_history",
    [
        (3, 0, 2, 5, 10, True, [0.0, 0.0, 3.3333333333333335]),
        (30, 0, 10, 20, 10, True, [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0,
                                   8.0, 9.0, 10.0, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10])
    ]
)
```

Reply:
@fco-dv you can do this : 
```python
config1 = (3, 0, 2, 5, 10, True, [0.0, 0.0, 3.3333333333333335])
expected_hist2 = [0.0] * 10 + [float(i) for i in range(1, 11)] + [10.0] * 10
config2 = (30, 0, 10, 20, 10, True, expected_hist2)

@pytest.mark.parametrize(
    "max_epochs, initial_value, step_constant, step_max_value, max_value, save_history, expected_param_history",
    [config1, config2],
)
def test_linear_scheduler_linear_increase_history(
    max_epochs, initial_value, step_constant, step_max_value, max_value, save_history, expected_param_history
):
    # Testing linear increase
```